sips/validators_per_operator_1k_cap.md

```diff
@@ -0,0 +1,61 @@
+|     Author     |               Title               | Category  |       Status        |    Date    |
+| -------------- | --------------------------------- | --------- | ------------------- | ---------- |
+| Matheus Franco | 1K cap on validators per operator | Contracts | open-for-discussion | 2024-11-28 |
+
+## Summary
+
+This SIP proposes increasing the cap on the number of validators per operator from 500 to 1000.
+
+## Motivation
+
+The recent optimizations allowed operators to reduce their operational costs while running the same number of validators, thus keeping the same profits. Therefore, this allows operators to further increase the size of their validator sets, keeping a sustainable processing cost and increasing their profitability.
+
+## Performance trade-offs
+
+To understand the performance impact, we compared two settings with the same number of validators (namely, 104k), one with a cap of 500 and the other of 1000 validators per operator. Using the 500 case as a benchmark, we got the following results for the 1000 case:
+
+| Metric                             | Impact     |
+|------------------------------------|------------|
+| Maximum operator cryptography cost | 20% higher |
+| Average operator cryptography cost | 80% higher |
+| Maximum operator message rate      | 30% lower  |
+| Average operator message rate      | 22% lower  |
+
+This behavior is expected. More validators per operator reduce the number of required operators and allow Committee Consensus to aggregate more duties. This promotes a reduction in the number of exchanged messages and non-committee message ratios. On the other hand, since the partial signature phase is the heaviest influence on the cryptography cost, increasing the cap from 500 to 1k inevitably increases the cryptography cost.
+
+> [!NOTE]
+> With the Pectra fork, dropping the `CommitteeIndex` field in the attestation beacon object will allow an operator to verify all post-consensus signatures in constant time. In other terms, currently, due to different `CommitteeIndex`s, the complexity of signatures verification on the post-consensus phase is $\mathcal{O}(d)$, where $d$ is the number of duties. After the Pectra fork, it will become $\mathcal{O}(1)$, significantly reducing the overall cryptography cost.
+
+## Security
+
+Post-consensus messages may increase their average and maximum sizes as shown in the [spec changes](#partialsignaturemessages). Nonetheless, it's still way smaller than a consensus proposal message. Thus, the proposed change doesn't introduce extra concerns on network security.
```

Comment:
Is this security? I think performance

Reply:
This is security in the sense of DoS by large messages